api/envoy/config/endpoint/v3/endpoint_components.proto

```diff
@@ -121,7 +134,14 @@ message LocalityLbEndpoints {
   core.v3.Locality locality = 1;
 
   // The group of endpoints belonging to the locality specified.
-  repeated LbEndpoint lb_endpoints = 2;
+  repeated LbEndpoint lb_endpoints = 2
+      [(udpa.annotations.field_migrate).oneof_promotion = "lb_config"];
```

Comment:
I don't think it's legal to have a repeated field in a oneof, so this oneof_promotion won't actually work. :(

Reply:
Yes you are correct, thanks!
I think the right way would be to support both `lb_endpoints` and the `leds_cluster_locality_config` (similar to what's being done for VHDS). I've looked at my implementation and it won't require much change.

Reply:
Alternatively, consider deprecating the existing `lb_endpoints` field and replacing it with a message that contains the repeated field, and then putting *that* message in the oneof.  In other words, add this message:

```
message LbEndpoints {
  repeated LbEndpoint lb_endpoints = 1;
}
```

And then change the fields here to:

```
// Deprecated -- use inline_endpoints instead.
repeated LbEndpoint lb_endpoints = 2 [deprecated = true];

oneof endpoints {
  LbEndpoints inline_endpoints = 7;
  LedsClusterLocalityConfig leds_cluster_locality_config = 8;
}
```